### PR TITLE
fix(hz): Default json tag alignment protoc-gen-go

### PR DESCRIPTION
**Pull request by Hanson**
a9b8e1b4-5fb5-4d3e-b26a-e7ecc9ea96a9

json name should be snake like form and query

**What type of PR is this?**
fix

**Check the PR title.**
 
- [x] This PR title match the format: <type>(optional scope): <description>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

(Optional) Translate the PR title into Chinese.
修复 json 默认不是下划线的问题

**(Optional) More detail description for this PR(en: English/zh: Chinese).**

**Which issue(s) this PR fixes:**

### CODE REVIEW

File: cmd/hz/internal/protobuf/tags.go [hunk 1]
```diff
@@ -258,7 +258,7 @@ func m2s(mt model.Tag) (ret [2]string) {
 
 func reflectJsonTag(f protoreflect.FieldDescriptor) (ret model.Tag) {
 	ret.Key = "json"
-	ret.Value = checkSnakeName(f.JSONName())
+	ret.Value = checkSnakeName(f.Name())
```
Comment (li-jin-gou) [inline cmd/hz/internal/protobuf/tags.go:261]: I think you need to fill in the pr title and description according to the pull request template.

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

File: cmd/hz/internal/protobuf/tags.go (hunk 1 above)
Comment (Hanson) [reply cmd/hz/internal/protobuf/tags.go:261]: done

**Review by li-jin-gou — APPROVED** [review]

**Comment by FGYFFFF** [comment]
Change the title to "Default json tag alignment protoc-gen-go" instead of "snake style". @Hanson

**Comment by Hanson** [comment]
> Change the title to "Default json tag alignment protoc-gen-go" instead of "snake style". @Hanson

done

**Pull request merged**